.github/workflows/build_and_deploy.yml

```diff
@@ -194,35 +224,42 @@ jobs:
       with:
         workflow: Smoke Test
         token: ${{ secrets.GIT_HUB_SERVICE_ACCOUNT_TOKEN }}
-        inputs: '{"paas_environment": "${{ env.ENVIRONMENT }}", "sha": "${{ github.sha }}"}'
-        ref: ${{ needs.build.outputs.build-tag}}
+        inputs: '{"paas_environment": "${{ env.ENVIRONMENT }}", "sha": "${{ env.COMMIT_SHA }}"}'
+        ref: ${{ env.BUILD_GIT_TAG }}
 
     - name: Wait for smoke test
       id: wait_for_smoke_test
       uses: fountainhead/action-wait-for-check@v1.0.0
       with:
         token: ${{ secrets.GIT_HUB_SERVICE_ACCOUNT_TOKEN }}
-        checkName: Smoke Test ${{ env.ENVIRONMENT }} ${{ github.sha }}
-        ref: ${{ needs.build.outputs.build-tag}}
+        checkName: Smoke Test ${{ env.ENVIRONMENT }} ${{ env.COMMIT_SHA }}
+        ref: ${{ env.BUILD_GIT_TAG }}
         timeoutSeconds: 300
         intervalSeconds: 15
 
     - name: Exit whole workflow if ${{ env.ENVIRONMENT }} smoke test was not successful
       if: steps.wait_for_smoke_test.outputs.conclusion != 'success'
       run: exit 1
 
+    - name: Post sticky pull request comment
+      if: github.event_name == 'pull_request'
```

Comment:
does this work? as the workflow doesn't get triggered on a `pull_request` trigger?

Reply:
Yes it is triggered here: https://github.com/DFE-Digital/teaching-vacancies/pull/3570/files/ecc0e9c8faaa5c9c1ebcd92869a226795726f45e#diff-8812abff9ed11e2e98d5066031a3181c6d44b28aea00f0d369532d5cb254650fR14

And you can see the successful runs for this PR

Reply:
ignore this, missed the `pull_request` trigger